### ISSUE TITLE
Error creating mixin config mixins.cardinal_components_item.json on startup

### ISSUE THREAD

**Issue by SchnitzelTim** [Ladysnake/Cardinal-Components-API#142]
I am trying to make a mods-updated version with CurseForge of the Sky FABRICation 2 mod pack just for myself and I get a RuntimeException from the cardinal-components-item mod on Minecraft on startup:
[latest.log](https://github.com/OnyxStudios/Cardinal-Components-API/files/9945598/latest.log)

How can I fix this?
I have set the profile to only list Release versions of the mods and there is only one Release version of the mod.

**Comment by Pyrofab** [comment]
Minecraft 1.17 huh ? We don't actively support this version anymore, but try installing one of the betas if the release doesn't work ?

**Comment by SchnitzelTim** [comment]
When I restarted Minecraft, the error disappeared. Eventually, it needed to create that json file, that it missed and rethrew the exception or so. I then had errors from other mods that the Java version was not 17. I then tried to go down to an older fabric version and then got incompatibility errors from even other mods. I finally, gave up and play the original mod pack.
Thanks for for your answer.

I would now close this ticket with status acceptable which does not exist, it seems.
